elasticsearch/index_templates/com.redhat.viaq-openshift-operations.template.json

```diff
@@ -943,7 +934,11 @@
   },
   "order": 10,
   "settings": {
-    "index.refresh_interval": "5s"
+    "index.refresh_interval": "5s",
+    "index.number_of_replicas": $REPLICA_SHARDS,
```

Comment:
FYI @lukas-vlcek - added this in here since we could no longer specify `index` settings in the elasticsearch config.
We do substitution of these values in `run.sh` before we push them into ES

Reply:
@ewolinetz I moved all the `settings.*` into extra json file. See my PR https://github.com/openshift/origin-aggregated-logging/pull/922

Going forward we could maintain the `settings.*` json files in this repo and keep the pure mappings in common data model repo. Pure separation... WDYT?
//cc @richm @jcantrill

Reply:
Yes, that sounds good to me.